.github/workflow.yaml

```
@@ -0,0 +1,142 @@
+    branches:
+      - main
+      - develop
+      - staging
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      
```

yamllint

[error] trailing spaces

(trailing-spaces)